### PR TITLE
Select DDP package only when required

### PR DESCRIPTION
**Pull request by Jaganathancse**
This change is to run DDP package selection only
when user provided dpm_ddp_package is different
than default value.

### CODE REVIEW

File: playbooks/select_kernel_ddp_package.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+- name: Select Kernel DDP Package
+  hosts: all
+  strategy: linear
+  become: true
+  vars:
+    edpm_ddp_package: 'ddp'
```
Comment (rebtoor) [inline playbooks/select_kernel_ddp_package.yml:7]: why pass the `edpm_ddp_package` variable with its default value?

**Review by rebtoor — CHANGES_REQUESTED** [review]

**Comment by rebtoor** [comment]
recheck

File: playbooks/select_kernel_ddp_package.yml (hunk 1 above)
Comment (Jaganathancse) [reply playbooks/select_kernel_ddp_package.yml:7]: @rebtoor just provided variable initialization with default value to use this playbook.
I Agree, need to remove this section to use playbook without changing. will update the code.

**Review by rebtoor — APPROVED** [review]

**Comment by rebtoor** [comment]
recheck

**Comment by rebtoor** [comment]
recheck

due to older failed job: https://logserver.rdoproject.org/36/336/ead1b893768200d1de0f1125d5bad0dcbe1e91a6/github-check/openstack-k8s-operators-content-provider/576571a/ci-framework-data/logs/ci_make_006_openstack-operator_-_call_catalog-build.log

**Comment by rebtoor** [comment]
recheck

There was a transient network error while collecting logs after podman role molecule test:

```
2023-09-26 14:57:20.509362 | LOOP [fetch-output : Collect logs, artifacts and docs]
2023-09-26 14:57:21.146439 | controller | changed:
2023-09-26 14:57:21.146672 | controller | .d..t...... ./
2023-09-26 14:57:21.146705 | controller | >f+++++++++ ansible-execution.log
2023-09-26 14:57:21.146730 | controller | >f+++++++++ installed-pkgs.log
2023-09-26 14:57:21.146753 | controller | >f+++++++++ python.log
2023-09-26 14:57:21.146776 | controller | >f+++++++++ registries.conf
2023-09-26 14:57:21.146799 | controller | >f+++++++++ report.html
2023-09-26 14:57:21.146821 | controller | >f+++++++++ selinux-denials.log
2023-09-26 14:57:21.146843 | controller | cd+++++++++ system-config/
2023-09-26 14:57:21.146864 | controller | cd+++++++++ system-config/libvirt/
2023-09-26 14:57:21.146886 | controller | >f+++++++++ system-config/libvirt/libvirt-admin.conf
2023-09-26 14:57:21.146907 | controller | >f+++++++++ system-config/libvirt/libvirt.conf
2023-09-26 14:57:21.146942 | controller | changed: All items complete
2023-09-26 14:57:21.146968 | 
2023-09-26 14:57:21.583909 | controller | changed: .d..t...... ./
2023-09-26 14:57:21.826193 | controller | ERROR: Item: {'dest': '/var/lib/zuul/builds/36a35cf4414d4e35a61d68696296e1f0/work/docs', 'src': 'docs'}
2023-09-26 14:57:21.826416 | controller | {
2023-09-26 14:57:21.826465 | controller |   "ansible_loop_var": "zj_output",
2023-09-26 14:57:21.826533 | controller |   "msg": "kex_exchange_identification: read: Connection reset by peer\r\nrsync: connection unexpectedly closed (0 bytes received so far) [Receiver]\nrsync error: unexplained error (code 255) at io.c(226) [Receiver=3.1.3]\n",
2023-09-26 14:57:21.826569 | controller |   "rc": 255,
2023-09-26 14:57:21.826602 | controller |   "zj_output": {
2023-09-26 14:57:21.826633 | controller |     "dest": "/var/lib/zuul/builds/36a35cf4414d4e35a61d68696296e1f0/work/docs",
2023-09-26 14:57:21.826664 | controller |     "src": "docs"
2023-09-26 14:57:21.826694 | controller |   }
2023-09-26 14:57:21.826724 | controller | }
2023-09-26 14:57:21.852171 | 
2023-09-26 14:57:21.852324 | PLAY RECAP
2023-09-26 14:57:21.852375 | controller | ok: 11 changed: 7 unreachable: 0 failed: 1 skipped: 8 rescued: 0 ignored: 3
```

**Pull request merged**